src/Message/AbstractOrderRequest.php

```diff
@@ -58,12 +57,12 @@ protected function getOrderData()
 
         /** @var Address $shippingAddress */
         if (null !== $shippingAddress = $this->getShippingAddress()) {
-            $data['shipping_address'] = $this->getAddress($shippingAddress);
+            $data['shipping_address'] = (array) $shippingAddress;
```

Comment:
Why does this have an array cast, rather than the `->getArrayCopy()` used for the billing address?

Reply:
oops was a test :D